### PR TITLE
Updated Replex container with new features and removed deprecated

### PR DESCRIPTION
**Pull request by PilaScat**

### CODE REVIEW

**Comment by nwithan8** [comment]
User `sarendsen` has changed their GitHub username to `lostb1t`. Currently, GitHub simply redirects the old links, but this will ensure no 404s in the future.

**Comment by PilaScat** [comment]
> User `sarendsen` has changed their GitHub username to `lostb1t`. Currently, GitHub simply redirects the old links, but this will ensure no 404s in the future.

While I was there I changed that too

File: templates/replex.xml [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-  <Config Name="Redirect Streams Endpoint" Target="REPLEX_REDIRECT_STREAMS_URL" Default="" Description="Alternative streams endpoint" Type="Variable" Display="always" Required="false" Mask="false"/>
-  <Config Name="Cache TTL" Target="REPLEX_CACHE_TTL" Default="900" Description="Time to live for caches in seconds. Set to 0 to disable (not recommended)" Type="Variable" Display="always" Required="false" Mask="false">900</Config>
-  <Config Name="Auto-refresh Recommended Rows Cache?" Target="REPLEX_CACHE_REFRESH" Default="true" Description="If set to true, the cache for recommended rows are auto-refreshed for each user" Type="Variable" Display="always" Required="false" Mask="false">true</Config>
+  <Config Name="Plex Host" Target="REPLEX_HOST" Default="http://localhost:32400" Description="URL of your Plex instance" Type="Variable" Display="always" Required="true" Mask="false">http://localhost:32400</Config>
+  <Config Name="Plex Token" Target="REPLEX_TOKEN" Default="" Description="Plex Token need for hero images" Type="Variable" Display="always" Required="true" Mask="false">
```
Comment (Roxedus) [inline templates/replex.xml:15]: This tag doesn't get closed

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/replex.xml (hunk 1 above)
Comment (PilaScat) [reply templates/replex.xml]: Only the required one needs to be closed? Or all of them?

**Comment by PilaScat** [comment]
I also looked at other templates and the guide, however it gives me errors when it shouldn't

**Comment by Roxedus** [comment]
It is still not valid XML, run it trough https://xmllint.com/

**Comment by PilaScat** [comment]
Ready to merge

**Review by Roxedus — APPROVED** [review]

**Pull request merged**